models/experimental/functional_yolov4/tt/ttnn_downsample2.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import torch
+import torch.nn as nn
+
+from ttnn.model_preprocessing import preprocess_model
+
+import ttnn
+import tt_lib
+from models.experimental.functional_yolov4.tt.ttnn_resblock import TtResBlock
+
+
+class TtDownSample2:
```

Comment:
please attached the ttnn graph generated my running the module to this PR so we can ensure it matches with the torch graphs.